client/system.go

```diff
@@ -50,3 +51,17 @@ func (client *APIClient) SystemInfo(ctx context.Context) (*types.SystemInfo, err
 
 	return info, err
 }
+
+// RegistryLogin requests a registy server to login
```

Comment:
Please add a period at the end of the sentence. Otherwise, this will be not in the godoc I think. @Ace-Tang